src/components/DrumElement.tsx

```
@@ -0,0 +1,106 @@
+}
+
+export const DrumElement = ({ src, style, onClick, note, type = 'tom' }: ElementProps) => {
+  const [isHit, setIsHit] = useState(false)
+
+  const handleMouseDown = () => {
+    onClick()
+    setIsHit(true)
+    setTimeout(() => setIsHit(false), 300)
+  }
+
+  const onKeyDown = (event: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)